src/cascadia/TerminalSettingsEditor/Appearances.xaml

```diff
@@ -210,6 +212,10 @@
                               IsEnabled="{x:Bind UsingMonospaceFont, Mode=OneWay}" />
                 </StackPanel>
             </local:SettingContainer>
+            <muxc:InfoBar x:Uid="Profile_FontFace_ProportionalFontWarning"
+                          IsOpen="{x:Bind ShowProportionalFontWarning, Mode=OneWay}"
+                          Severity="Warning"/>
+

```

Comment:
either the above extra newline, or a `runxamlformat` to do xaml code formatting. Those would be my guess